filament/src/PostProcessManager.cpp

```diff
@@ -915,12 +916,17 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::dof(FrameGraph& fg,
     auto depth = blackboard.get<FrameGraphTexture>("depth");
     assert_invariant(depth);
 
-    // the downsampled target is multiple of 8, so we can have 4 clean mipmap levels
-    constexpr const uint32_t maxMipLevels = 4u;
+    /*
+     * dofResolution is used (at compile time for now) to chose between full- or quarter-resolution
+     * for the DoF calculations. Set to [1] for full resolution or [2] for quarter-resolution.
+     */
+    const uint32_t dofResolution = 2;
```

Comment:
If this were a #define then you could use it to swap between dofCoc and dofDownsample in sMaterialList, which would allow the binary size to stay as small as possible.

Reply:
True -- but I was thinking of possibly making it a runtime option, e.g. for a photo/very high quality mode.